src/diffusers/schedulers/scheduling_heun_discrete.py

```diff
@@ -70,8 +70,6 @@ class HeunDiscreteScheduler(SchedulerMixin, ConfigMixin):
             `linear` or `scaled_linear`.
         trained_betas (`np.ndarray`, optional):
             option to pass an array of betas directly to the constructor to bypass `beta_start`, `beta_end` etc.
-            options to clip the variance used when adding noise to the denoised sample. Choose from `fixed_small`,
-            `fixed_small_log`, `fixed_large`, `fixed_large_log`, `learned` or `learned_range`.
```

Comment:
Is this an unrelated change?

Reply:
Yes, based on this [comment](https://github.com/huggingface/diffusers/pull/3020#discussion_r1162245750), it wasn't correct in Heun as well. I can split it out to a different PR, if that's better?

Reply:
I think the change is small enough to be accommodated here. @yiyixuxu WDYT?

Reply:
agreed - no need to open a new PR :)